src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    <p className="Person__partner">
+      {(person.sex === 'm' && person.isMarried === true) && `${person.partnerName} is my wife`}
+      {(person.sex === 'f' && person.isMarried === true) && `${person.partnerName} is my husband`}
+      {person.isMarried === false && `I am not married`}
```

Comment:
Use ternary operator to render person's status or "I am not married" message
https://react.dev/learn/conditional-rendering#conditional-ternary-operator--